### PR TITLE
Update vendors.yaml to include Cribl

### PR DESCRIPTION
**Pull request by nheudecker**
Adds Cribl as an OTEL vendor, including documentation link.

### CODE REVIEW

**Comment by svrnm** [comment]
@nheudecker please sign the CLA, also make sure that you fill out the contact field (github handle is ideal, email address is possible, we will use this to notify vendors about change)

**Comment by nheudecker** [comment]
@svrnm CLA signed. Thanks!

File: data/ecosystem/vendors.yaml [hunk 1]
```diff
@@ -41,6 +41,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: Cribl
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:44]: @nheudecker please add Cribl in line 1 to the list of words that should be ignored by the spell checker

Comment (nheudecker) [reply data/ecosystem/vendors.yaml:44]: Looks like I somehow added Cribl successfully.

**Review by svrnm — APPROVED** [review]

File: data/ecosystem/vendors.yaml [hunk 2]
```diff
@@ -41,6 +41,13 @@
...
+  distribution: false
+  nativeOTLP: true
+  url: 'https://docs.cribl.io/stream/sources-otel'
+  contact: ''
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:48]: please add a point of contact for us (ideally a github handle, can be an email address)

Comment (nheudecker) [reply data/ecosystem/vendors.yaml]: Updated with contact information - hello@cribl.io

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
Thank you @nheudecker

**Pull request merged**